### DISCUSSION TITLE
[A-L] (2023/24) Foglio 1 - Esercizio 4

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#159] [Esercizi - Viaggi/Piazza]
![es 4](https://github.com/Elia-Belli/algebra2324/assets/107944476/33135fe6-cf43-458f-9542-951d177d5765)

**Comment by Elia-Belli** [comment]
Dimostriamo le 3 proprietà di una relazione di equivalenza:
- **Riflessività**: per ipotesi e definizione di relazione d'eq. $\forall a \in A, (a,a) \in R_1,R_2$
$\Rightarrow \forall a \in A, (a,a) \in R_1 \cap R_2$
- **Simmetria**: per ipotesi e definizione di relazione d'eq:
$\forall (a,b) \in R_1 \Rightarrow (b,a) \in R_1$
$\forall (a,b) \in R_2 \Rightarrow (b,a) \in R_2$
$\Rightarrow (a,b) \in R_1 \cap R_2 \Rightarrow (b,a) \in R_1 \cap R_2$
- **Transitività**: per ipotesi e definizione di relazione d'eq:
se $(a,b) \land (b,c) \in R_1 \Rightarrow (a,c) \in R_1$
se $(a,b) \land (b,c) \in R_2 \Rightarrow (a,c) \in R_2$
$\Rightarrow (a,b) \land (b,c) \in R_1 \cap R_2 \Rightarrow (a,c) \in R_1 \cap R_2$

**Reply by FeddyLix17** [reply]
mi trovo completamente in linea con questa dimostrazione, in particolare proporrei le seguenti modifiche 
(sostanzialmente aggiungere degli ∀)

> ⇒ (a, b) ∈ R1 ∩ R2 ⇒ (b, a) ∈ R1 ∩ R2
```diff
-  ⇒  (a, b) ∈ R1 ∩ R2 ⇒ (b, a) ∈ R1 ∩ R2
+  ⇒ ∀(a, b) ∈ R1 ∩ R2 ⇒ (b, a) ∈ R1 ∩ R2
```

> se (a, b) ∧ (b, c) ∈ R1 ⇒ (a, c) ∈ R1
```diff
- se (a, b) ∧ (b, c) ∈ R1 ⇒ (a, c) ∈ R1
+  ∀ (a, b) ∧ (b, c) ∈ R1 ⇒ (a, c) ∈ R1
```
> se (a, b) ∧ (b, c) ∈ R2 ⇒ (a, c) ∈ R2
```diff
- se (a, b) ∧ (b, c) ∈ R2 ⇒ (a, c) ∈ R2
+  ∀ (a, b) ∧ (b, c) ∈ R2 ⇒ (a, c) ∈ R2
``` 

> (a, b) ∧ (b, c) ∈ R1 ∩ R2 ⇒ (a, c) ∈ R1 ∩ R2
```diff
-   (a, b) ∧ (b, c) ∈ R1 ∩ R2 ⇒ (a, c) ∈ R1 ∩ R2
+ ∀ (a, b) ∧ (b, c) ∈ R1 ∩ R2 ⇒ (a, c) ∈ R1 ∩ R2 
```

**Comment by Elia-Belli** [comment]
@CarloDaRomadev attento a scrivere $\forall (x,y) \in A$ nella simmetria e transitività, queste sono proprietà che valgono per le coppie presenti in $R_1$ e $R_2$, d'altronde $A$ è un insieme di singoletti e non di tuple.

**Comment by daromacarlo** [comment]
@Elia-Belli grazie per la correzione.
![Immagine 2023-10-08 193311](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/ba8c0e69-ac65-41ff-a4ca-9eb23b503e1a)
così è corretto?

**Comment by Frisbon** [comment]
![image_2023-10-11_10-49-51](https://github.com/sapienzastudentsnetwork/algebra2324/assets/84731341/57cf0c02-c293-4f10-88b3-45b536c92245)

Spero sia giusto lmao
